package/setup.py

```diff
@@ -295,6 +296,15 @@ def extensions(config):
     else:
         encore_compile_args.append('-O3')
 
+    # allow using custom c/c++ flags and architecture specific instructions.
+    # This allows people to build optimized versions of MDAnalysis.
+    # Do here so not included in encore
```

Comment:
Can you add a short version of this to the setup.cfg comment too please? (i.e. that it doesn't affect encore compilation flags)

Reply:
Yep for sure.